### PR TITLE
Adding local collection points

### PR DESCRIPTION
**Pull request by thorbjornsen**
Added local collection points based on a specified site/region value. The default (no value specified) will work exactly like the previous implementation. This change will not break current implementations that use the plugin.

### CODE REVIEW

**Review by huylenq — APPROVED** [review]
Looks good to me, I'm not super familiar with these but the change looks straightforward.

**Comment by huylenq** [comment]
On second thought, justify from what's going on in TD iOS and Android SDKs, I highly recommend making the endpoint configurable the way `TDApiKey` and `TDDatabase` are. Unlikely there's a need to change endpoint at runtime, and we would have a single way to set things up. WDYT?
Also, JFYI, it's up to you but on iOS and Android SDKs, endpoints are simply strings, they don't use predefined enums.

**Comment by thorbjornsen** [comment]
Made the modification to get the region from the config file.

**Review by huylenq — APPROVED** [review]
LGTM

File: Source/TreasureData/Private/TreasureDataProvider.h [hunk 1]
```diff
@@ -97,4 +107,26 @@ class FAnalyticsProviderTreasureData :
...
+	}
+
+	inline FString GetAPIURL()
+	{
+		switch (GetRegion())
+		{
+		case US02:
+			return TEXT("https://in.treasuredata.com/postback/v3/event/");
+		case AP01:
+			return TEXT("https://tokyo.in-development.treasuredata.com/postback/v3/event/");
```
Comment (huylenq) [inline Source/TreasureData/Private/TreasureDataProvider.h]:
```suggestion
			return TEXT("https://tokyo.in.treasuredata.com/postback/v3/event/");
```

Comment (thorbjornsen) [reply Source/TreasureData/Private/TreasureDataProvider.h]: HAHA! Good find.. lemme fix that

**Review by huylenq — APPROVED** [review]
2425939 also looks good to me.

**Pull request merged**